ReactWindows/ReactNative.Net46/Touch/TouchHandler.cs

```diff
@@ -383,6 +429,58 @@ public override void Dispatch(RCTEventEmitter eventEmitter)
             }
         }
 
+        class PointerEnterExitEvent : Event
+        {
+            private readonly TouchEventType _touchEventType;
+
+            public PointerEnterExitEvent(TouchEventType touchEventType, int viewTag)
+                : base(viewTag, TimeSpan.FromTicks(Environment.TickCount))
+            {
+                _touchEventType = touchEventType;
+            }
+
+            public override string EventName
+            {
+                get
+                {
+                    return _touchEventType.GetJavaScriptEventName();
+                }
+            }
+
+            public override bool CanCoalesce
+            {
```

Comment:
please collapse simple properties like this to more concise syntax:
public override bool CanCoalesce { get; } = false;